daemon/launch_unix.go

```diff
@@ -244,6 +248,19 @@ func LaunchDaemons(ctx context.Context, launchers []Launcher, egrp *errgroup.Gro
 					metrics.SetComponentHealthStatus(metrics.HealthStatusComponent(metricName), metrics.StatusCritical,
 						launchers[chosen].Name()+" process failed unexpectedly")
 					err = errors.Wrapf(waitResult, "%s process failed unexpectedly", launchers[chosen].Name())
+					service_key := "origin"
+					if config.IsServerEnabled(server_structs.CacheType) {
+						service_key = "cache"
+					}
+					crashTimestamp := time.Now().Unix()
+					log.Debug("Recording xrootd crash at time: ", crashTimestamp)
```

Comment:
Right now this records a unix timestamp, which isn't very friendly to read in a log. Can you instead convert the value into an RFC3339-conformant string?